src/compressed_tensors/quantization/utils/helpers.py

```diff
@@ -155,24 +162,36 @@ def compute_dynamic_scales_and_zp(value: Tensor, args: QuantizationArgs):
         reduced dimensions
     :return: tuple of scale and zero point derived from the observed tensor
     """
+
+    keep_dims = True
     if args.strategy == QuantizationStrategy.TOKEN:
         dim = {1, 2}
         reduce_dims = tuple(idx for idx in range(value.ndim) if idx not in dim)
     elif args.strategy == QuantizationStrategy.TENSOR:
         reduce_dims = None
+    elif args.strategy == QuantizationStrategy.TENSOR_GROUP:
+        # per group dynamic quantization - only valid for
+        # activations
+        dim = {0, 1}
+        value = value.squeeze(0)
+        reduce_dims = tuple(idx for idx in range(3) if idx not in dim)
+        keep_dims = False
+        value = torch.reshape(
+            value, (value.shape[0], value.shape[1] // args.group_size, args.group_size)
```

Comment:
Technically, this should be a ceil, or at least include an assertion that non-dividing group sizes are not supported by vllm

Reply:
We can also have an assert if we don't want to deal with that right now